static/css/wall_display.css

```
@@ -160,6 +160,55 @@ html {
+  0%, 100% { opacity: 1; }
+  50% { opacity: 0.3; }
+}
+.rp-field-aux {
+  flex-direction: row;
+  align-items: center;
+  font-size: 16px;
+  color: #fff;
+}
+.rp-label-aux {
+  font-size: 11px;
+  font-family: "FuturaLT";
```

Stylelint (17.4.0)

[error] Unexpected quotes around "FuturaLT" (font-family-name-quotes)

(font-family-name-quotes)